src/resolve/resolved-bus.c

```diff
@@ -2115,6 +2125,7 @@ static const sd_bus_vtable resolve_vtable[] = {
         SD_BUS_PROPERTY("DNSSECNegativeTrustAnchors", "as", bus_property_get_ntas, 0, 0),
         SD_BUS_PROPERTY("DNSStubListener", "s", bus_property_get_dns_stub_listener_mode, offsetof(Manager, dns_stub_listener_mode), 0),
         SD_BUS_PROPERTY("ResolvConfMode", "s", bus_property_get_resolv_conf_mode, 0, 0),
+        SD_BUS_PROPERTY("Monitor", "b", bus_property_get_monitor_enabled, 0, 0),
```

Comment:
This should definitely send a signal when it changes, hence last parameter should be `SD_BUS_VTABLE_PROPERTY_EMITS_CHANGE`

Reply:
Fixed in latest patch